src/app/shared/entity/provider/anexia.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export class AnexiaVlan {
+  id: string;
```

Comment:
This structure is a bit strange, is it implemented that way in the API?

cc @zreigz

Reply:
I mean that we are using the object to store just one property. Wouldn't it be better to make it a string array in that case? Or is it like this because there will be more properties?

Reply:
It is implemented this way in the api atm

Reply:
Okay, I will not block you with it. Perhaps we will adjust it in the future.